playground/src/components/details/exercise_detail/text.tsx

```diff
@@ -7,9 +7,7 @@ type TextExerciseDetailProps = {
   exercise: TextExercise;
 };
 
-export default function TextExerciseDetail({
-  exercise,
-}: TextExerciseDetailProps) {
+export default function TextExerciseDetail({ exercise }: TextExerciseDetailProps) {
```

Comment:
I think this is clearer:
```suggestion
export default function TextExerciseDetail({ exercise }: { exercise: TextExercise }) {
```
(and removing the `TextExerciseDetailProps` above)